### PR TITLE
Update remote-help.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a pre requisite per issue#https://github.com/MicrosoftDocs/memdocs/issues/2939

### CODE REVIEW

**Comment by PRMerger13** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/remote-actions/remote-help.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ The Remote help app supports the following capabilities:
 - Remote help add-on license for all IT support workers (helpers) and users (https://aka.ms/PremiumAddOnsDocs)
 - Windows 10/11
 - The remote help app for Windows. See [Install and update remote help](#install-and-update-remote-help)
+-  Remote help will be available for M365 A1/A3/A5 Education customers later this calendar year. Education customers with M365 A1/A3/A5 which include Intune subscription will have Remote Help with their license and they won’t require the **Remote Help Add-on**.
```
Comment (JohanFreelancer9) [inline memdocs/intune/remote-actions/remote-help.md:75]:
```suggestion
-  Remote help will be available for Microsoft 365 A1/A3/A5 Education customers later this calendar year. Education customers with Microsoft 365 A1/A3/A5 which include Intune subscription will have remote help with their license and they won’t require the **Remote Help Add-on**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Dear @msbemba. I have done a copy review, but I am not too sure about including this prerequisite. The latest guidance is that we don't make announcements in docs content and avoid talking about future events. Thanks.

**Comment by dougeby** [comment]
> Dear @msbemba. I have done a copy review, but I am not too sure about including this prerequisite. The latest guidance is that we don't make announcements in docs content and avoid talking about future events. Thanks.

I agree. We don't ever provide information about what will be available with particular licenses at a particular date even if it's something that was already announced. I'm also curious how the information was vetted for accuracy.

**Comment by msbemba** [comment]
@dougeby and @JohanFreelancer9  Thank you for that information. I'll close this PR. I viewed a Roadmap that said Rolled out (but in preview for Edu) so that was the confusion.  I'll ensure that I add information only after the product is live and do not commit when it is in roadmap.

image

Thanks
Sri

**Pull request closed without merging**